virt/virtual_machines/advanced_vm_management/virt-using-containerdisk-volumes.adoc

```diff
@@ -0,0 +1,18 @@
+[id="virt-using-containerdisk-volumes"]
+= Using containerDisk volumes with virtual machines
+include::modules/virt-document-attributes.adoc[]
+:context: virt-using-containerdisk-volumes
+toc::[]
+
+A containerDisk volume is a virtual machine image stored as a container in an image registry
```

Comment:
`A containerDisk volume is a virtual machine image stored inside a container image in a container image registry`

Reply:
Thanks mate